app/models/route.rb

```diff
@@ -3,4 +3,16 @@ class Route < ApplicationRecord
   has_many :bus_stops_routes
   has_many :bus_stops, through: :bus_stops_routes
   default_scope { order :number }
+
+  def stops_by_sequence(route_id, direction)
```

Comment:
These are both instance methods. Why do we pass the route ID around both methods?

Reply:
If we didn't we would have to call both methods in the controller. Alternatively I could make this one method.

Reply:
My point is that these are instance methods, and so you already know what the route is. It's `self`. You shouldn't need to pass it as an argument.

Reply:
Oh, I see.